lib/rules/key-spacing.js

```diff
@@ -347,6 +347,40 @@ module.exports = {
             );
         }
 
+        /**
+         * Starting from the given a node (a property.key node here) looks forward
+         * until it finds the colon punctuator and returns it.
+         * @param {ASTNode} node The node to start looking from.
+         * @returns {ASTNode} The colon punctuator.
+         */
+        function getNextColon(node) {
+            return sourceCode.getTokenAfter(node, astUtils.isColonToken);
+        }
+
+        /**
+         * Starting from the given a node (a property.key node here) looks forward
```

Comment:
Same here and in other places also.
```suggestion
         * Starting from the given node (a property.key node here) looks forward
```

Reply:
Thanks! Haven't noticed that in the copy-paste.